docs/src/ci.md

```diff
@@ -59,14 +59,397 @@ export default defineConfig({
 });
 ```
 
-
 ## CI configurations
 
+### GitHub Actions
+
 The [Command line tools](./browsers#install-system-dependencies) can be used to install all operating system dependencies on GitHub Actions.
```

Comment:
that should be not part of `### GitHub Actions` I think and moved one layer above.

Reply:
The sentence contains "GitHub Actions". If it applies to more CI providers, then we'll need to reword it.

Reply:
I'd reword, e.g. on your CI provider.